crm_security_group/views/menu_items.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="crm.crm_menu_root" model="ir.ui.menu">
+        <field
+            name="groups_id"
+            eval="[(4, ref('crm_security_group.group_crm_own_leads'))]"
+        />
+    </record>
+    <record id="sale_crm.sale_order_menu_quotations_crm" model="ir.ui.menu">
+        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
+    </record>
+    <record id="crm.sales_team_menu_team_pipeline" model="ir.ui.menu">
+        <field
+            name="groups_id"
+            eval="[(4, ref('crm_security_group.group_crm_manager'))]"
+        />
+    </record>
+    <record id="crm.crm_menu_report" model="ir.ui.menu">
+        <field
+            name="groups_id"
+            eval="[(4, ref('crm_security_group.group_crm_manager'))]"
+        />
+    </record>
+    <record id="crm.crm_menu_config" model="ir.ui.menu">
+        <field
+            name="groups_id"
+            eval="[(4, ref('crm_security_group.group_crm_manager'))]"
+        />
+    </record>
+    <!-- It's necessary to show sale menu only if user have sale group!-->
+    <record id="sale.sale_menu_root" model="ir.ui.menu">
+        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
```

Comment:
Not sure this one is really needed, as all the submenus already have a security group, and thus if no visible children, this one is not shown AFAIK.